### PR TITLE
First attempt at cleaning up the GUI code

### PR DESCRIPTION
**Pull request by tsujan**
(1) Removed htable2 and pstable2 source files (not used anywhere).

(2) Removed some redundant variables and comments.

(3) Removed the terrible functions `QtTableView::verticalScrollBar()` and `QtTableView::horizontalScrollBar()` (who knows why they were needed with Qt2?!).

(4) Corrected a mistake in drawing header sections (there are still other mistakes).

NOTE 1: I usually don't include several commits in a single PRs but, believe me, the code is a mess and the sooner it's cleaned up, the better.

NOTE 2: A huge amount of garbage still remains.

### CODE REVIEW

File: src/htable.cpp [hunk 1]
```diff
@@ -1031,16 +1018,8 @@ void HeadedTable::setTreeMode(bool tm)
...
-        int old_sorted = sorted_col;
-        sorted_col = col;
-        //		printf("old_sorted=%d\n",old_sorted);
-        //		printf("sorted_col=%d\n",sorted_col);
-        if (old_sorted != -1 && old_sorted < ncols) // ncols bug
-            updateHeading(old_sorted);
-        if (col != -1 && col < ncols)
-            updateHeading(col);
-    }
+    int old_sorted = sorted_col;
```
Comment (shlyakpavel) [inline src/htable.cpp]: old_sorted seems to be unused now

File: src/qttableview.cpp [hunk 2]
```diff
@@ -1040,18 +967,18 @@ void QtTableView::paintEvent(QPaintEvent *e)
     if (!colXPos(firstCol, &xStart))
     {
         // right empty area of table
-        // paint.eraseRect( updateR ); // erase area outside cells but
+        // p.eraseRect( updateR ); // erase area outside cells but
```
Comment (shlyakpavel) [inline src/qttableview.cpp]: Why changing the commented code?

Comment (tsujan) [reply src/qttableview.cpp]: Find-and-replace. Not important because there's a long long way to a clean code.

File: src/htable.cpp (hunk 1 above)
Comment (tsujan) [reply src/htable.cpp]: Like below: "Not important because there's a long long way to a clean code." This is only the first attempt.

**Comment by tsujan** [comment]
After this is merged, a class-by-class cleanup would be better. I've already done it for SearchBox, ControlBar and StatusBar as the next PR. Then, just a few hundred classes remain... piece of cake.

**Pull request merged**